compiler/prelude/primops.txt.pp

```diff
@@ -990,7 +992,9 @@
    number of elements to copy, copy the elements from the source array
    to the destination array. The source and destination arrays can
    refer to the same array. Both arrays must fully contain the
-   specified ranges, but this is not checked.}
+   specified ranges, but this is not checked.
+   The regions are allowed to overlap, although this is only possible when the same 
+   array is provided as both the source and the destination. }
```

Comment:
I feel like this wording is a tad confusing. Perhaps,
> In the case where the source and destination are the same array the source and destination regions may overlap
I don't feel strong one way or another though.